apps/system/js/core.js

```diff
@@ -110,11 +109,12 @@
         'MultiScreenController',
         'PinsManager',
         //IFDEF_FIREFOX_SYNC
-        'SyncManager'
+        'SyncManager',
         //ENDIF_FIREFOX_SYNC
         // XXX: We should move CarrierInfoNotifier into mobileConnectionCore,
         // but integration tests running on desktop without mobileConnection
         // is testing this.
+        'CustomModules'
```

Comment:
I guess this means statusbar only loads when idle. Are we sure that is what we want for the smartphone boot?